src/Workspaces/Core/Portable/Utilities/LazyInitialization.cs

```diff
@@ -1,17 +1,19 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+#nullable enable
+
 using System;
 using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
 using System.Threading;
 
 namespace Roslyn.Utilities
 {
     internal static class LazyInitialization
     {
-        internal static T InterlockedStore<T>(ref T target, T value) where T : class
+        internal static T InterlockedStore<T>(ref T? target, T value) where T : class
         {
-            Debug.Assert(value != null);
-            return Interlocked.CompareExchange(ref target, value, null) ?? value;
+            return Interlocked.CompareExchange(ref target, value, null!) ?? value;
```

Comment:
😕 Why `null!`?

Reply:
Hmm, at some point the compiler reported a warning. Now I tried removing it and it works ...

Reply:
I'm guessing the change from `ref T` to `ref T?` fixed it.